g2p/app.py

```diff
@@ -213,6 +214,14 @@ async def convert(sid, message):
         )
         return
     transducer = CompositeTransducer(transducers)
+    try:
+        in_lang = message["data"]["mappings"][0]["kwargs"]["in_lang"]
+        tokenizer = make_tokenizer(in_lang)
+        transducer = TokenizingTransducer(transducer, tokenizer)
+    except Exception:
```

Comment:
Someday, a linter may complain about this.  I see that there are a couple of distinct things that could go wrong in the try block, but as you say, tokenization is optional - the problem would arise if it starts to fail unexpectedly and we won't know exactly why.